libraries/joomla/document/jsonapi.php

```diff
@@ -41,6 +41,7 @@ public function __construct($options = array())
 
 		// Set mime type to JSON-API
 		$this->_mime = 'application/vnd.api+json';
+		$this->_type = 'jsonapi';
```

Comment:
We need to decide on whether to make this type explicit or not as it defines what the View Format is (i.e. Should component's be limited to a single JSON view or should be have json+ld and jsonapi and json views all available?